### DISCUSSION TITLE
What should be the ideal block-size

### DISCUSSION THREAD

**Discussion by ayush-san** [juicedata/juicefs#1094] [Q&A]
Hi,

I am using juicefs as persistent storage for jupyterhub docker notebooks. Currently, many users have complained that while training the model they have seen performance degradation. Upon debugging I found that juicefs process metrics on that node spiked. I am assuming this is due to small block-size, or can there be another param which I need to tune? 

Here's the screenshot of juicefs metrics 
![image](https://user-images.githubusercontent.com/57655135/144562072-e4e83e38-7941-4317-b756-61ee9cf52b08.png)
![image](https://user-images.githubusercontent.com/57655135/144562234-22728eab-af45-4254-8824-f31fcf0dc85f.png)

**Comment by xiaogaozi** [comment]
@ayush-san Hi, you could use `juicefs profile` to analyze which operation is too slow, please refer to [the document](https://juicefs.com/docs/community/operations_profiling) for more information.
